app/tasks/brief_tasks.py

```diff
@@ -0,0 +1,15 @@
+from . import celery
+from app import db
+from app.models import Brief
+from app.emails.briefs import send_brief_closed_email
+import pendulum
+
+
+@celery.task
+def process_closed_briefs():
+    # find briefs that were closed yesterday. this task is designed to run after midnight.
+    closed_briefs = db.session.query(Brief).filter(Brief.closed_at >= pendulum.yesterday(),
```

Comment:
Referring to my comment on the scheduled time config - as the Celery beat schedule is in UTC, I believe you may need to specify `tz='Australia/Sydney'` or similar here for the `.yesterday()` and `.today()` calls to work as you'd expect, as I believe our prod environment is running this code in UTC. For example, if you aim for a 2AM AEST scheduled time (4PM UTC), then `.today(tz='Australia/Sydney')` should be 2 hours ago, where as `.today(tz='UTC')` is 16 hours ago (because these pendulum functions default to 00:00:00, as opposed to say 23:59:59).